lib/modules/showImages.js

```diff
@@ -1185,6 +1210,14 @@ function generateMedia(options: ExpandoMedia, context: {| href: string |}) {
 			throw new Error(`Unreachable: invalid media type ${options.type}`);
 	}
 
+	if (options.crosspostData) {
+		try {
+			element.prepend(crosspostMetadataTemplate(options.crosspostData));
+		} catch (e) {
+			console.error(`showImages: could not create crosspost metadata expando for ${context.href}`, e);
```

Comment:
Is this reachable? At a glance it doesn't appear that anything in `crosspostMetadataTemplate` can throw

Reply:
It might fail if Reddit changes the data attributes.

Reply:
How though? ``` string.html` "${null}" "${undefined}" "${false}" ` === ` "" "" "" ` ```, and all the property accesses are one level deep, so it seems like it would just render nothing instead of throwing?